### DISCUSSION TITLE
common point stacking method question

### DISCUSSION THREAD

**Discussion by sunzhe-96** [gmtsar/gmtsar#279] [General]
Dear all,
Hello,ereyone,I have a question,Why is the deviation of the deformation rate after and before I use common point stacking method for the same area too large? first picture is after atm method,second is before atm method
![after](https://user-images.githubusercontent.com/89780038/154795247-c05c523e-4eac-42b1-bb43-f97ab458eb5d.JPG)
![before](https://user-images.githubusercontent.com/89780038/154795252-41a5241e-7f2f-4529-af97-fb5b9865bdda.JPG)

**Comment by Xiaohua-Eric-Xu** [comment]
Hi @sunzhe-96 , you probably want to plot them to the same color range. My suspicion is that certain pixels' decorrelation noise (your area looks pretty noisy) got magnified through the CSS atm correction.

**Reply by sunzhe-96** [reply]
> Hi @sunzhe-96 , you probably want to plot them to the same color range. My suspicion is that certain pixels' decorrelation noise (your area looks pretty noisy) got magnified through the CSS atm correction.

Thank you Eric，I probably understand，I‘ll try plot them to the same color range,by the way,I'm a gmt novice, do you have any solution to this problem and thank you so much for your patience.

**Reply by Xiaohua-Eric-Xu** [reply]
You could `gmt makecpt -Cturbo -T-30/50/10 -Z -D > dr_plot.cpt`, then `grdimage -Cdr_plot.cpt` to use the new cpt file. Later you need to use `colorbar` or `psscale` that utilize that cpt file for plotting the colorscale.

**Reply by sunzhe-96** [reply]
> You could `gmt makecpt -Cturbo -T-30/50/10 -Z -D > dr_plot.cpt`, then `grdimage -Cdr_plot.cpt` to use the new cpt file. Later you need to use `colorbar` or `psscale` that utilize that cpt file for plotting the colorscale.

Thank you so much Eric! I'll try!

**Reply by sunzhe-96** [reply]
I am so sorry to bother you again,I use `-Cdr_plot.cpt`  that has been running without results and can run the results without using the dr_plot.cpt
![colorbar](https://user-images.githubusercontent.com/89780038/155051769-a78f4306-64a6-4821-9031-61dcf8e85987.JPG)
![colorbar2](https://user-images.githubusercontent.com/89780038/155051775-281b4435-ef68-4ff6-91e0-4d18235efc68.JPG)

**Reply by Xiaohua-Eric-Xu** [reply]
So you are using modern mode. I am not particularly sure. Do you need to add -Cdr_plot.cpt to colorbar as well?

**Reply by Xiaohua-Eric-Xu** [reply]
Also, did you generate dr_plot.cpt somewhere outside the script?

**Reply by sunzhe-96** [reply]
So,what should I add -Cdr_plot.cpt to colorbar? and I generate dr_plot.cpt in folder just like picture
![colorbar3](https://user-images.githubusercontent.com/89780038/155060256-1cb70c68-a90d-4c33-8cc7-c9b1896d78ea.JPG)

**Reply by sunzhe-96** [reply]
Hello,Eric,I changed the script and it works, sorry to bother you with this basic problem.
`gmt begin atm_a png
    gmt makecpt -Cturbo -T-30/50/10 -Z -D
    gmt grdimage vel2_atm.grd -JX6i+ -Q -B
    gmt colorbar -Bxaf+l"deformatin rate(mm/y)"
 gmt end`
